app/contexts/layout-context.tsx

```diff
@@ -0,0 +1,61 @@
+import { createContext, useContext } from 'react';
+
+type LayoutContextType = {
+  // isSettingsOpen: SettingsContextType,
+  // isChatOpen: ChatContextType,
+  isParticipantsListOpen: ParticipantsListContextType;
+};
+
+export const CustomLayoutContext = createContext<LayoutContextType | undefined>(undefined);
+
+export function useCustomLayoutContext(): LayoutContextType {
+  const customLayoutContext = useContext(CustomLayoutContext);
+  if (!customLayoutContext) {
+    throw Error('Tried to access LayoutContext context outside a LayoutContextProvider provider.');
+  }
+  return customLayoutContext;
+}
+
+interface CustomLayoutContextProviderProps {
+  layoutContextValue: LayoutContextType;
+  children: React.ReactNode;
+}
+
+export function CustomLayoutContextProvider({
+  layoutContextValue,
+  children,
+}: CustomLayoutContextProviderProps) {
+  return (
+    <CustomLayoutContext.Provider value={layoutContextValue}>
+      {' '}
+      {children}{' '}
```

Comment:
I think the linter may have auto added some of these, not sure if they're needed?
```suggestion
      {children}
```